src/app/app-routing.module.ts

```diff
@@ -21,6 +22,10 @@ const routes: Routes = [
     path: 'employee/show/:id',
```

Comment:
As mentioned earlier, please try to make it more `DRY`, all routes paths should be on one file.